pipeline/lib/metric.ml

```diff
@@ -0,0 +1,100 @@
+module Github = Current_github
+
+module MetricKey = struct
+  type t = string * string * string
+
+  let compare a b = compare a b
```

Comment:
This `MetricKey` is a very very good idea! :) (for this kind of comparison, I like to do `let compare = Stdlib.compare` to make it obvious + not run into issues if another `compare` enters the scope)